styles/styles.css

```diff
@@ -1188,6 +1189,10 @@ main .section.padded-inline .section-container {
     padding-inline: 0;
   }
 
+  main .section.full-width.columns-container > .section-container {
```

Comment:
If this is columns specific style, probably it should go in `columns.css`. BTW, full-width implies using to full width and this seem to be restricting the width. Does removing `full-width` from the section in the word doc or splitting up the section help ?

Reply:
Understood. Actually if I remove full-with from the section it's the same outcome so I will just remove this selector.

Reply:
Here is the page to check: 

https://ht--sunstar--hlxsites.hlx.page/healthy-thinking